### ISSUE TITLE
How to use Huggingface pytorch bert to generate the prediction TSV file from the test set of a GLUE task?

### ISSUE THREAD

**Issue by hrheru20** [huggingface/transformers#3692]
# ❓ Questions & Help

## Details

Hello folks! Can you provide some simple example of using pytorch bert to generate the prediction TSV file from the test set of a GLUE task (such as MRPC) based on the fine-tuned model so that I can submit the prediction TSV file for each GLUE task to GLUE leaderboard? Thank you very much.

**A link to original question on Stack Overflow**:

**Comment by julien-c** [comment]
This was implemented ~1 month ago so closing this issue.
